### PR TITLE
Add useful ExtCal functions

### LINKED ISSUE #544: Not all functions with ExtCal in it should be omitted from the Python APIs
Sure, external calibration is not supported. But in functions_addon.py for the drivers we tend to mark all functions containing "ExtCal" as 'codegen_method':'no'.

Upon manual inspection of the NI-FGEN API, I realized this is too broad. Some functions work from a regular session and are valuable.

Here are all the functions we have that contain ExtCal. ( @texasaggie97 : Note that this doesn't yet include NI-SCOPE which isn't in master). Some of these should be included in the Python APIs and called from the system tests.

```
kirsch0:nimi-python kirsch$ find * | grep functions\.py | xargs grep -ir \'.*extcal.*\':
src/nidcpower/metadata/functions.py:    'ChangeExtCalPassword': {
src/nidcpower/metadata/functions.py:    'CloseExtCal': {
src/nidcpower/metadata/functions.py:    'GetExtCalLastDateAndTime': {
src/nidcpower/metadata/functions.py:    'GetExtCalLastTemp': {
src/nidcpower/metadata/functions.py:    'GetExtCalRecommendedInterval': {
src/nidcpower/metadata/functions.py:    'InitExtCal': {
src/nidmm/metadata/functions.py:    'CloseExtCal': {
src/nidmm/metadata/functions.py:    'GetExtCalRecommendedInterval': {
src/nidmm/metadata/functions.py:    'InitExtCal': {
src/nidmm/metadata/functions.py:    'RestoreLastExtCalConstants': {
src/nifgen/metadata/functions.py:    'ChangeExtCalPassword': {
src/nifgen/metadata/functions.py:    'CloseExtCal': {
src/nifgen/metadata/functions.py:    'GetExtCalLastDateAndTime': {
src/nifgen/metadata/functions.py:    'GetExtCalLastTemp': {
src/nifgen/metadata/functions.py:    'GetExtCalRecommendedInterval': {
src/nifgen/metadata/functions.py:    'InitExtCal': {
src/nifgen/metadata/functions.py:    'RestoreLastExtCalConstants': {
```

These we should add:

* GetExtCalLastDateAndTime
* GetExtCalLastTemp
* GetExtCalRecommendedInterval

### PR DESCRIPTION
**Pull request by marcoskirsch**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

[X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

[X] I've added tests applicable for this pull request

### What does this Pull Request accomplish?

Adds"ext_cal" methods that are useful in a regular session, not only in a calibration session (calibration not supported in Python APIs).

### List issues fixed by this Pull Request below, if any.

* Fix #544 

### What testing has been done?

Added system tests.

### CODE REVIEW

**Comment by marcoskirsch** [comment]
coverage/coveralls is due to merge from master, not related to this PR

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ bin/
 commands.log
 .vscode/launch.json
 .vscode/settings.json
+*_system_test_result.json
```
Comment (texasaggie97) [inline .gitignore]: Isn't this in bin/ now?

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +28,7 @@ All notable changes to this project will be documented in this file.
   * #### Removed
 * ### NI-DCPower
   * #### Added
+    * get_ext_cal_recommended_interval()
```
Comment (texasaggie97) [inline CHANGELOG.md]: You only have on listed here but it looks like more were added.

**Comment by marcoskirsch** [comment]
> What about other drivers? Different PR? Not applicable?

```
Kirsch0:nimi-python kirsch$ find src | grep functions\.py | xargs grep -ir \'.*ExtCal.*\'\:
src/nidcpower/metadata/functions.py:    'ChangeExtCalPassword': {
src/nidcpower/metadata/functions.py:    'CloseExtCal': {
src/nidcpower/metadata/functions.py:    'GetExtCalLastDateAndTime': {
src/nidcpower/metadata/functions.py:    'GetExtCalLastTemp': {
src/nidcpower/metadata/functions.py:    'GetExtCalRecommendedInterval': {
src/nidcpower/metadata/functions.py:    'InitExtCal': {
src/nidmm/metadata/functions.py:    'CloseExtCal': {
src/nidmm/metadata/functions.py:    'GetExtCalRecommendedInterval': {
src/nidmm/metadata/functions.py:    'InitExtCal': {
src/nidmm/metadata/functions.py:    'RestoreLastExtCalConstants': {
src/nifgen/metadata/functions.py:    'ChangeExtCalPassword': {
src/nifgen/metadata/functions.py:    'CloseExtCal': {
src/nifgen/metadata/functions.py:    'GetExtCalLastDateAndTime': {
src/nifgen/metadata/functions.py:    'GetExtCalLastTemp': {
src/nifgen/metadata/functions.py:    'GetExtCalRecommendedInterval': {
src/nifgen/metadata/functions.py:    'InitExtCal': {
src/nifgen/metadata/functions.py:    'RestoreLastExtCalConstants': {
```

I had already done NI-FGEN, it's how I caught this.

**Pull request merged**